pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -190,6 +187,24 @@ func NewFit(_ context.Context, plArgs runtime.Object, h fwk.Handle, fts feature.
 type ResourceRequestsOptions struct {
 	EnablePodLevelResources   bool
 	EnableDRAExtendedResource bool
+	DRAManager                fwk.SharedDRAManager
```

Comment:
DRAManager  is not a ResourceRequestsOptions, so it seems weird to put it here. I can see it's convenient to have it here, so it can be passed around without fewer changes to isFit -> Fits -> fitRequest signature. I'll leave it to @macsko to decide. But at least add some comments to explain why it is here.

Reply:
Addressed